cmd/promtool/main.go

```diff
@@ -52,6 +52,7 @@ func main() {
 		"config-files",
 		"The config files to check.",
 	).Required().ExistingFiles()
+	checkConfigSyntaxOnly := checkConfigCmd.Flag("syntax-only", "Only check the config file syntax; ignore certificate and token checks").Bool()
```

Comment:
I wouldn't mention certs or tokens (which come from a library in any case). It's files and their contents.

Reply:
Thanks, I'll change the wording.